### PR TITLE
"Using the ide" Page [Part 2]

### PR DESCRIPTION
**Pull request by JR1811**
# Information

This PR is a continuation of https://github.com/FabricMC/fabric-docs/pull/111.

The Fork branch needed to be cleaned due to an error on my side, so here is the fresh PR with basically the same commits.

In addition, I also committed the changes suggested by @modmuss50 from back in the old PR (moved page to misc group)

### CODE REVIEW

File: develop/ide-tips-and-tricks.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+authors: 
+    - JR1811
+---
+
+# IDE Tips and Tricks{#ide-tips-and-tricks}
+
+This page gives useful bits of information, to speed up and ease the workflow of developers. Incorporate them into yours, to your liking.
+It may take some time to learn and get used to the shortcuts and other options. You can use this page as a reference for that.
+
+::: note
```
Comment (IMB11) [inline develop/ide-tips-and-tricks.md]:
```
::: warning
```

**Review by IMB11 — CHANGES_REQUESTED** [review]
I think this is ready to merge, just need you to fix this one callout block. Only `info`, `warning` , `error`, `tip` and `details` are supported.

**Comment by JR1811** [comment]
By the way, I've noticed that the vidstack video embed's thumbnails are massively zoomed in. Is there a way to fix it?
If not or it's too bothersome, it is good enough for me.

grafik

**Comment by IMB11** [comment]
> By the way, I've noticed that the vidstack video embed's thumbnails are massively zoomed in. Is there a way to fix it? If not or it's too bothersome, it is good enough for me.
> 
> grafik

I think it's actually the first frame of the video, there's not much we can do about it, but it's a huge improvement over plain youtube iframes in terms of privacy and performance.

**Review by IMB11 — APPROVED** [review]

**Pull request merged**